client/src/components/Buttons/Button.notifications.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { IconButton, NotificationsIcon } from 'constants/icons';
+
+export const ButtonNotifs = () => {
+  return (
+    <IconButton
```

Comment:
Given this seems to have similar styling in several places, maybe it should have that as default with `variant="light"` kind of option? Then this file may not be needed and you could inline pretty cleanly, and make it easier to make global changes to styles of like-buttons